front_end/models/react_native/ReactDevToolsBindingsModel.ts

```diff
@@ -223,6 +263,7 @@ export class ReactDevToolsBindingsModel extends SDK.SDKModel.SDKModel {
           return new Promise(resolve => setTimeout(resolve, 250)).then(() => this.waitForFuseboxDispatcherToBeInitialized(attempt + 1));
         }
 
+        this.fuseboxDispatcherIsInitialized = true;
```

Comment:
Would it be simpler if we move this setter outside the invocation? Ideally colocated with flushing the queue, e.g.

```
this.fuseboxDispatcherIsInitialized = false;
void this.waitForFuseboxDispatcherToBeInitialized()
  .then(() => {
    this.dispatchEventToListeners(Events.BackendExecutionContextCreated);
    this.flushOutDomainMessagesQueues();
    this.fuseboxDispatcherIsInitialized = true;
  })
```

Reply:
> Would it be simpler if we move this setter outside the invocation?

Then you have to rely on the fact that it is set after each invocation, not sure if this is simpler